content/_data/changelogs/lts.yml

```diff
@@ -140,6 +140,77 @@
       message: Do not fail to write a log file just because something deleted the parent directory.
       issue: 16634
       pull: 2738
+- version: "2.46.1"
+  date: 2017-04-19
+  lts_predecessor: "2.32.3"
+  lts_baseline: "2.46"
+  lts_changes:
+    - type: rfe
+      message: Update the SSHD module from 1.7 to 1.8. The change disables obsolete Ciphers: AES128CBC, TripleDESCBC, and BlowfishCBC.
```

Comment:
This "message" should be quoted, otherwise it's not valid YAML (due to the colons fwiw)

```
An error occurred: (/home/jenkins/workspace/Infra_jenkins.io_PR-764-NYRWJGUU4XY76C35W6WKURUQQGJVESLRGXZL7GGWN3MFTWWRZL7Q/content/_data/changelogs/lts.yml): mapping values are not allowed here at line 149 column 92
org/jruby/ext/psych/PsychParser.java:227:in `parse'
```